lib/modelManagers/followModel.js

```diff
@@ -0,0 +1,117 @@
+const { follows, User } = require('../../database/models');
+/**
+ * @description Follow and Unfollow user
+ *
+ * @class FollowModelManager
+ */
+class FollowModelManager {
+  /**
+   * @description Get a single User
+   * @static
+   * @param {any} id
+   * @returns {Object} - return user
+   *
+   * @memberOf FollowModelManager
+   */
+  static async getUserById(id) {
```

Comment:
Why should you have a `getUserById` by id in `FollowModelManager`? This defeats the purpose of having a user model manager. I believe there is a method already in the user model manager that performs this task.

Reply:
Fixed